scipy/stats/_continuous_distns.py

```diff
@@ -3501,12 +3497,23 @@ def _pdf(self, x, mu):
     def _logpdf(self, x, mu):
         return -0.5*np.log(2*np.pi) - 1.5*np.log(x) - ((x-mu)/mu)**2/(2*x)
 
+    def _logcdf(self, x, mu):
+        fac = np.sqrt(1 / x)
+        a = _norm_logcdf(fac * (x - mu) / mu)
+        b = 2 / mu + _norm_logcdf(-fac * (x + mu) / mu)
+        return a + np.log1p(np.exp(b - a))
```

Comment:
All this looks good. On one hand, I appreciate that it uses the variable names/factoring that we already had.
On the other hand, it makes it harder to compare to the new reference (but I did check it). 
On the third hand, the new reference has "r-project" in the URL, and that is often concerning since R code is typically GPL (not license compatible) - but you did not refer to any R code, just the equations on page 342 of the reference. (right?)
(No action required here. Just recording my thoughts.)